examples/scenes/src/lib.rs

```diff
@@ -43,11 +43,11 @@ pub struct ExampleScene {
 }
 
 pub trait TestScene {
-    fn render(&mut self, sb: &mut SceneBuilder, params: &mut SceneParams);
+    fn render(&mut self, sb: &mut Scene, params: &mut SceneParams);
```

Comment:
`sb` here is short for `SceneBuilder`, which is of course no longer accurate

I'm not certain what we should change it to. Maybe `scene` would be fine?

Reply:
Oops, good catch. Renamed all of these (and occurrences of `builder`) to `scene`.